contracts/src/core/interfaces/IEigenDAThresholdRegistry.sol

```diff
@@ -47,3 +47,11 @@ interface IEigenDAThresholdRegistry {
     /// @notice Returns the blob params for a given blob version
     function getBlobParams(uint16 version) external view returns (DATypesV1.VersionedBlobParams memory);
 }
+
+interface IEigenDAThresholdRegistry is IEigenDAThresholdRegistryBase {
```

Comment:
Why is this separation between Base and notBase needed? Base doesn't seem to be a Base given that its got both v1 and v2 stuff, and then this interface is adding more V2 stuff on top. Pretty confusing. Can we describe the interface and why its organized this way?

Reply:
Is that goal that once we get rid of V1, then `Base` will be deleted and we'll only be left with this minimal `IEigenDAThresholdRegistry` interface? Why does it need to extend the Base interface btw? All good things to document

Reply:
No, the goal here is simply to accommodate the fact that the ServiceManager should not be upgraded, but right now it inherits the Threshold Registry. It is a mess.